lib/internal/modules/esm/loader.js

```diff
@@ -115,9 +114,9 @@ class Loader {
         /^data:([^/]+\/[^;,]+)(?:[^,]*?)(;base64)?,/,
         url,
       );
-      throw new ERR_INVALID_MODULE_SPECIFIER(
-        url,
-        dataUrl ? `has an unsupported MIME type "${dataUrl[1]}"` : '');
+      throw new ERR_UNKNOWN_MODULE_FORMAT(
```

Comment:
This is a breaking change (ex Mocha listens for this). How do we mark that for an experimental feature?